nixos/modules/services/networking/unifi.nix

```diff
@@ -110,27 +118,77 @@ in
       '';
     };
 
+    httpPort = mkOption {
+      type = types.ints.u16;
```

Comment:
There is a `types.port` type which was since the last release you can use here.

Reply:
Great, didn't know that.